index.js

```diff
@@ -42,6 +42,67 @@ function set(obj, path, value) {
   obj[components[components.length - 1]] = value;
 }
 
+function find(list, predicate) {
+  // 1. Let O be ? ToObject(list value).
+  if (list == null) {
+    throw new TypeError('"list" is null or not defined');
+  }
+
+  var o = Object(list);
+
+  // 2. Let len be ? ToLength(? Get(O, "length")).
+  var len = o.length >>> 0;
+
+  // 3. If IsCallable(predicate) is false, throw a TypeError exception.
+  if (typeof predicate !== 'function') {
+    throw new TypeError('predicate must be a function');
+  }
+
+  // 4. If thisArg was supplied, let T be thisArg; else let T be undefined.
```

Comment:
Probably meant to be "If arguments was supplied, "..

Reply:
The code is lifted from MDN's definition for Array.find() - I'll clean it up a little, but in the future will need to do the ES6 upgrade to the code as per #4